CMakeLists.txt

```diff
@@ -10,8 +10,9 @@ include(cmake/install-prefix.cmake)
 include(cmake/wsl.cmake)
 include(cmake/simd.cmake)
 
-set(CMAKE_C_STANDARD 11)
```

Comment:
If this is a requirement for an external library, it should be defined there (another example of why a package manager would be very beneficial).